vita3k/renderer/src/gl/screen_render.cpp

```diff
@@ -94,6 +94,9 @@ void ScreenRenderer::render(const SceFVector2 &viewport_pos, const SceFVector2 &
     GLboolean last_enable_depth = glIsEnabled(GL_DEPTH_TEST);
     GLboolean last_enable_cull = glIsEnabled(GL_CULL_FACE);
 
+    GLint last_polygon_mode;
```

Comment:
```suggestion
    GLint last_polygon_mode[2];
```

See https://stackoverflow.com/questions/37115829/opengl-switching-polygon-mode-causes-segfault